LION/classical_algorithms/sirt.py

```diff
@@ -0,0 +1,71 @@
+from typing import Callable, Optional
+import torch
+from LION.CTtools.ct_geometry import Geometry
+from LION.CTtools.ct_utils import make_operator
+from ts_algorithms import sirt as ts_sirt
+import tomosipo as ts
+
+from LION.exceptions.exceptions import NoDataException
+
+
+def sirt_from_geo(
```

Comment:
Maybe interesting to discuss: 

One of the advantages of not doing static typing is that we can have 1 SIRT only. Classically in satic typing lagnuages you can overload functions, but the advantage of dynamically typed ones like python is that instead of having something like `sirt_from_geo` we can simply do:
```
assert isinstance(geo,Geometry) or isinstance(geo,ts.Operator.Operator), seconed input must be a ct.Geometry or an operator. 
if isinstance(geo,Geometry):
    op=make_operator(geo)
```

opinions?

Reply:
also works! I don't really prefer one over the other, to be honest, I suppose it's a question of consistency amongst methods.

Reply:
one other thing I'd say is if we do something like that, it's adding an extra check every time where we may not need it. From memory isinstance uses reflection , which is quite slow...